docs/general/how-to-copy-the-contents-of-an-open-xml-package-part-to-a-document-part-in-a-dif.md

```diff
@@ -110,7 +110,7 @@ is stored in the ZIP item theme/theme1.xml:
 ```
 
 
-© ISO/IEC29500: 2008.
+© [!include[ISO/IEC 29500 version](../includes/iso-iec-29500-version.md)]
```

Comment:
```suggestion
© [!include[ISO/IEC 29500](../includes/iso-iec-29500-version.md)]
```

same as above - it seems like a consistent `ISO/IEC 29500` would be ideal

Reply:
not sure why the actual version is its own include, since either way something will have to be updated once the next version of 29500 is published.

Reply:
> not sure why the actual version is its own include, since either way something will have to be updated once the next version of 29500 is published.

If it is it's own snippet, we will only need to update it once instead of in a lot of different files.